waspc/src/Wasp/Project/WebApp.hs

```diff
@@ -17,3 +17,9 @@ getBaseDir :: AppSpec -> String
 getBaseDir spec = fromMaybe "/" maybeBaseDir
   where
     maybeBaseDir = AS.App.Client.baseDir =<< AS.App.client (snd $ getApp spec)
+
+defaultClientPort :: Int
+defaultClientPort = 3000
+
+getDefaultClientUrl :: AppSpec -> String
+getDefaultClientUrl spec = "http://localhost:" ++ show defaultClientPort ++ getBaseDir spec
```

Comment:
Why extract this? Are there other places that assume this and we should also maybe extract it?

Reply:
I wanted to construct the `defaultClientUrl` in Haskell and not piece it together in the template. 

That required the extraction of the default port as well.